src/bp/ui-shared/src/Form/FormFields/Switch/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Switch } from '@blueprintjs/core'
+import React, { FC } from 'react'
+
+import style from './style.scss'
+
+const SharedSwitch: FC<any> = ({ fieldKey, label, className, checked, onChange, children }) => {
```

Comment:
Ah I think I was too lazy to rename the import from blueprint, so the different name to prevent collision haha. Well, no special reason for a switch, I just prefer this visually. But a checkbox could be fine also. I don't really have a preference actually